task/redhat-dependency-analytics/0.2/redhat-dependency-analytics.yaml

```diff
@@ -0,0 +1,147 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+
+metadata:
+  name: redhat-dependency-analytics
+  labels:
+    app.kubernetes.io/version: "0.2"
+  annotations:
+    tekton.dev/categories: Security
+    tekton.dev/pipelines.minVersion: "0.37.5"
+    tekton.dev/tags: Security, Vulnenrability, CVE
+    tekton.dev/displayName: "Red Hat Dependency Analytics"
+    tekton.dev/platforms: "linux/amd64"
+spec:
+  description: >-
+    The Red Hat Dependency Analytics task is an interface between Tekton and Red Hat Dependency Analytics (RHDA) platform.
+    It provides vulnerability and compliance analysis for application dependencies in your software supply chain.
+
+  workspaces:
+    - name: output
+      description: Volume backing this workspace is used for input/output of the task.
+
+  params:
+    - name: manifest-file-path
+      description: Path to target manifest file within workspace.
+
+    - name: output-file-path
+      description: Path to file within workspace, where the analysis report is saved.
+      default: redhat-dependency-analytics-report.json
+
+    - name: rhda-image
+      description: Image where Exhort Javascript API and required dependencies are installed.
+      default: quay.io/ecosystem-appeng/exhort-javascript-api:0.1.1-ea.26
+
+    - name: python-image
+      description: Image with installed Python interpreter and associated tools (such as pip, pip3, etc.).
+      default: python:3.11
```

Comment:
can we use full path of the image?

Reply:
As the python image attribute is meant to provide the user with a dynamic solution where the user can choose the desired python version, it would be preferable to keep the image in its short form to show that any image format is acceptable. 
For tests though, i can surely provide a full path with a digest instead of tag.